pandas/io/json/table_schema.py

```diff
@@ -103,6 +108,40 @@ def make_field(arr, dtype=None):
     return field
 
 
+def revert_field(field):
```

Comment:
can we rename make_field and this to something more descriptive. make 
convert_json_typename_to_pandas (and the reverse)